hw7/Advanced_Task_1.ts

```diff
@@ -0,0 +1,18 @@
+// Advanced
+// Task 1
+// Реализуйте функцию которая будет проверять, является ли слово палиндромом.
+
+function isPalindrom(data: string): string {
+    if (data.split("").reverse().join("") == data) {
+        console.log("Это палиндром")
+    } else {
+        console.log("Это не палиндром")
+    }
+    return (data)
```

Comment:
Why should we return data here?

Reply:
Function had a string type, so it should return smth.
I changed to void type and removed return.
Hope this can be considered as fixed.

Reply:
Yes, it was a fix, because you should either console.log or return a string. It you return the string you should console.log the function then. Now it looks good.